errors/errors.go

```diff
@@ -28,6 +28,8 @@ const (
 	ErrBasicAuthNotSupported  = "invalid DSN: basic auth not enabled"
 	ErrInvalidDSNMaxRows      = "invalid DSN: maxRows param is not an integer"
 	ErrInvalidDSNTimeout      = "invalid DSN: timeout param is not an integer"
+	ErrInvalidUseCloudFetch   = "invalid format: useCloudFetch param is not a boolean"
```

Comment:
Let's add a helper function for invalid param values error messages.  That way they'll all be consistent.

Reply:
We talked offline, implementing a helper function directly accessed when throwing the error